data_structures/src/chain.rs

```diff
@@ -1363,19 +1363,21 @@ impl RADTally {
     }
 }
 
-type WeightedHash = (u64, Hash);
-type WeightedVTTransaction = (u64, VTTransaction);
+type PrioritizedHash = (u64, Hash);
```

Comment:
This `u64` is `nanowits / weight`, which looks like it should be a `f64`, but we cannot use floats as keys because they cannot be sorted. There is this crate that provides sortable floats, if we decide to use `f64` instead of `u64`:

https://docs.rs/ordered-float/1.0.2/ordered_float/struct.OrderedFloat.html